linux_pipeline/Jenkinsfile_upstream_kernel

```diff
@@ -16,7 +13,7 @@ def reportStageStatus(stageName, stageStatus) {
                                     variable: 'PERF_DB_CONFIG')]) {
         dir('kernel_version_report' + env.BUILD_NUMBER + env.BRANCH_NAME) {
               unstash 'kernel_version_ini'
-              sh returnStatus: true, script: '''
+              sh returnStatus: false, script: '''
```

Comment:
if this fails to report a passed stage the pipeline will stop. maybe set it to true and print a warning message in case of failure?

Reply:
will be changed